src/models/pats.rs

```diff
@@ -85,8 +85,24 @@ bitflags::bitflags! {
         // perform analytics action
         const PERFORM_ANALYTICS = 1 << 30;
 
-        const ALL = 0b1111111111111111111111111111111;
-        const NOT_RESTRICTED = 0b00000011111111111111100111;
+        // create a collection
+        const COLLECTION_CREATE = 1 << 31;
+        // read a user's collections
+        const COLLECTION_READ = 1 << 32;
+        // write to a collection
+        const COLLECTION_WRITE = 1 << 33;
+        // delete a collection
+        const COLLECTION_DELETE = 1 << 34;
+        // read a collection's files
+        const COLLECTION_FILES_READ = 1 << 35;
```

Comment:
what is this scope used for? shouldn't it just be bundled in with COLLECTION_READ